x-pack/plugins/cases/server/services/licensing.ts

```diff
@@ -6,15 +6,24 @@
  */
 
 import { firstValueFrom, Observable } from 'rxjs';
-import { ILicense, LicenseType } from '@kbn/licensing-plugin/server';
+import { ILicense, LicenseType, LicensingPluginStart } from '@kbn/licensing-plugin/server';
 
 export class LicensingService {
   private readonly license$: Observable<ILicense>;
+  private _notifyUsage: LicensingPluginStart['featureUsage']['notifyUsage'] | null = null;
 
   constructor(license$: Observable<ILicense>) {
     this.license$ = license$;
   }
 
+  public setNotifyUsage(notifyUsage: LicensingPluginStart['featureUsage']['notifyUsage']) {
```

Comment:
I believe if we remove this method and set the `notifyUsage` function in the constructor then `_notifyUsage` can be `readonly` and doesn't need to be `null` right?